v2/api-validator/tests/server-tests/collateral/account-link.test.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  CollateralAccount,
+  CollateralAccountLink,
+  CollateralLinkStatus,
+  CollateralSignerId,
+  CryptocurrencyReference,
+  Environment,
```

Comment:
Please rename this type everywhere, including in OpenAPI, to `AccountEnvironment`. It's too ambiguous otherwise.

Reply:
updated